.gitignore

```diff
@@ -0,0 +1,3 @@
+dist
+build
+*.swp
```

Comment:
I assume that `*.swp` is to prevent vim leavings ending up in the repo. this is better dealt with leveraging a global gitignore a la https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore